widgetastic_manageiq.py

```diff
@@ -1589,3 +1589,19 @@ class chart(TimelinesChart):  # NOQA
     @property
     def is_displayed(self):
         return self.title.text == 'Timelines'
+
```

Comment:
Obviously this isn't all the generic buttons, there's 'Save' and probably some more that we'll want, maybe for the search forms (load, delete)